lldb-eval/eval_test.cc

```diff
@@ -2224,3 +2238,22 @@ TEST_F(EvalTest, TestCompositeAssignmentBitwise) {
       EvalWithContext("$p >> $p", vars_),
```

Comment:
Not related to this change, but was this supposed to be `>>=`? There are also 5 other occurrences of `>>` or `<<` above in this test.

Reply:
Oh, whoops, that's a typo. Good catch!